tests/datasets/test_datasets.py

```diff
@@ -4,15 +4,19 @@
 
 @author: Lionel Massoulard
 """
+import os
+import shutil
+import tempfile
 
 import pytest
-from aikit.datasets.datasets import load_dataset, DatasetEnum
+from aikit.datasets.datasets import load_dataset, _load_public_path
 
 
-@pytest.mark.parametrize("name", DatasetEnum.alls)
+@pytest.mark.parametrize("name", ["titanic"])
 def test_load_dataset(name):
+    tempdir = tempfile.mkdtemp()
 
-    res = load_dataset(name)
+    res = load_dataset(name, cache_dir=tempdir)
 
```

Comment:
I think pytest has a fixture to for tmpdir
https://docs.pytest.org/en/latest/tmpdir.html

Reply:
https://github.com/societe-generale/aikit/pull/2/commits/e2729f328b7a3086a908ed270f82269173812cc8